spring-cloud-dependencies/pom.xml

```diff
@@ -183,6 +183,64 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>flatten-maven-plugin</artifactId>
+				<inherited>false</inherited>
+				<executions>
+					<execution>
+						<!-- Create an effective POM (with versions expanded) for the CLI and
```

Comment:
Do we need this one (copied from Boot I know). We don't use it in the CLI, or docs AFAIK.

Reply:
We don't really, let me remove it